Gems/Atom/RHI/DX12/Code/Source/RHI/Device.cpp

```diff
@@ -441,6 +513,38 @@ namespace AZ
             return MemoryView(resource.Get(), 0, allocationInfo.SizeInBytes, allocationInfo.Alignment, MemoryViewType::Image);
         }
 
+        MemoryView Device::CreateDx12maBuffer(
+            const RHI::BufferDescriptor& bufferDescriptor,
+            D3D12_RESOURCE_STATES initialState,
+            D3D12_HEAP_TYPE heapType)
+        {
+            D3D12_RESOURCE_DESC resourceDesc;
+            ConvertBufferDescriptor(bufferDescriptor, resourceDesc);
+
+#ifdef AZ_DX12_DXR_SUPPORT
+            if (initialState == D3D12_RESOURCE_STATE_RAYTRACING_ACCELERATION_STRUCTURE)
+            {
+                resourceDesc.Flags |= D3D12_RESOURCE_FLAG_ALLOW_UNORDERED_ACCESS;
+            }
+#endif
+
```

Comment:
We can refactor this part out with CreateBufferCommitted

Reply:
I'm not clear what you are suggesting here?